Xamarin.Forms.Platform.Android/Extensions/ImageViewExtensions.cs

```diff
@@ -18,6 +18,8 @@ internal static class ImageViewExtensions
 			ImageSource newImageSource,
 			ImageSource previousImageSource)
 		{
+			bool checkView = newView != null;
+			bool SourceIsNotChanged() => checkView ? newView?.Source == newImageSource : true;
```

Comment:
Can you change this to just take the parameters in so that it doesn't have to create a closure? Otherwise this looks good.